package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "uploadcare-netlifycms",
+  "version": "1.0.0",
+  "widgetVersion": "3.x",
+  "effectsTabVersion": "1.x",
+  "main": "dist/uploadcare-netlifycms.js",
+  "description": "This is Uploadcare plugin for [Netlify CMS].",
```

Comment:
File Uploader by Uploadcare. The plugin for NetlifyCMS to work with Uploadcare Widget.